### PR TITLE
[classifier] Bicycle charging station.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

File: data/mapcss-mapping.csv [hunk 1]
```diff
@@ -255,7 +255,7 @@ waterway|ditch|tunnel;[waterway=ditch][tunnel?];;name;int_name;254;
 deprecated|deprecated;255;x
 piste:type|downhill|intermediate;[piste:type=downhill][piste:difficulty=intermediate];;name;int_name;256;
 railway|subway_entrance;257;
-deprecated|deprecated;258;x
+amenity|charging_station|bicycle;[amenity=charging_station][bicycle?];;name;int_name;258;
```
Comment (pastk) [inline data/mapcss-mapping.csv:258]: 1. maybe `small_electric_vehicle=yes` should be considered same as `bicycle=yes`?
2. maybe make priority of `amenity-charging_station-motorcar` a little higher for charging stations with `[bicycle=yes],[motorcar=yes]`?

**Review by pastk — APPROVED** [review]

**Review by fitojb — APPROVED** [review]

File: data/categories.txt [hunk 2]
```diff
@@ -10705,16 +10705,16 @@ sk:4Veterinár
 fa:دامپزشکی
 mr:पशुवैद्यकीय डॉक्टर|प्राण्यांचा डॉक्टर
 
-amenity-charging_station
+@charging_station
 en:4Charging Station|charging
-ru:4Зарядная станция|станция зарядки|зарядка
-bg:4Зарядна станция|зареждане|електричество|електромобил
+ru:4Зарядная станция|зарядка
```
Comment (biodranik) [inline data/categories.txt:10710]:
```suggestion
ru:4Зарядная станция|зарядка|электрозарядка|зарядить
be:4Зарадная станцыя|зарадка|электразарадка|зарадзіць
```

File: data/categories.txt [hunk 3]
```diff
@@ -10705,16 +10705,16 @@ sk:4Veterinár
...
+bg:4Зарядна станция|зареждане|електричество
```
Comment (biodranik) [inline data/categories.txt:10711]:
```suggestion
bg:4Зарядна станция|зареждане
```

Comment (biodranik) [reply data/categories.txt:10711]: uk: добавить зарядка | електрозарядка | зарядити

File: data/mapcss-mapping.csv (hunk 1 above)
Comment (biodranik) [reply data/mapcss-mapping.csv:258]: 1. why?

File: data/categories.txt [hunk 4]
```diff
@@ -10741,6 +10741,16 @@ sk:Dobíjacia stanica|dobíjanie
 fa:ایستگاه شارژ
 mr:चार्जिंग स्टेशन|प्रभाराक केंद्र
 
+amenity-charging_station|@charging_station
+
+amenity-charging_station-bicycle|@charging_station
+en:4Bicycle Charging
+ru:4Велозарядка
```
Comment (biodranik) [inline data/categories.txt:10748]: Translations from string_types.txt should be copied here.

P.S. We should automate it.

**Review by biodranik — APPROVED** [review]

File: android/res/values-fr/strings.xml [hunk 5]
```diff
@@ -763,7 +763,8 @@
 	<string name="type.amenity.car_wash">Station de lavage</string>
 	<string name="type.amenity.casino">Casino</string>
 	<string name="type.amenity.charging_station">Borne de recharge</string>
-	<string name="type.amenity.charging_station.motorcar">Borne de recharge</string>
+	<string name="type.amenity.charging_station.bicycle">Station de recharge pour vélos</string>
+	<string name="type.amenity.charging_station.motorcar">Borne de recharge automobile</string>
```
Comment (Wivik) [inline android/res/values-fr/strings.xml:767]: Borne de recharge pour voiture

File: android/res/values-fr/strings.xml [hunk 6]
```diff
@@ -763,7 +763,8 @@
 	<string name="type.amenity.car_wash">Station de lavage</string>
 	<string name="type.amenity.casino">Casino</string>
 	<string name="type.amenity.charging_station">Borne de recharge</string>
-	<string name="type.amenity.charging_station.motorcar">Borne de recharge</string>
+	<string name="type.amenity.charging_station.bicycle">Station de recharge pour vélos</string>
```
Comment (Wivik) [inline android/res/values-fr/strings.xml:766]: Borne de recharge pour vélo

File: iphone/Maps/LocalizedStrings/fr.lproj/Localizable.strings [hunk 7]
```diff
@@ -1314,7 +1314,9 @@
 
 "type.amenity.charging_station" = "Borne de recharge";
 
-"type.amenity.charging_station.motorcar" = "Borne de recharge";
+"type.amenity.charging_station.bicycle" = "Station de recharge pour vélos";
```
Comment (Wivik) [inline iphone/Maps/LocalizedStrings/fr.lproj/Localizable.strings:1317]: Borne de recharge pour vélo

File: iphone/Maps/LocalizedStrings/fr.lproj/Localizable.strings [hunk 8]
```diff
@@ -1314,7 +1314,9 @@
...
+
+"type.amenity.charging_station.motorcar" = "Borne de recharge automobile";
```
Comment (Wivik) [inline iphone/Maps/LocalizedStrings/fr.lproj/Localizable.strings:1319]: Borne de recharge pour voiture

**Review by Wivik — COMMENTED** [review]
Updated French strings.

**Review by Wivik — APPROVED** [review]

**Review by d4f5409d — APPROVED** [review]

File: data/strings/types_strings.txt [hunk 9]
```diff
@@ -1050,8 +1050,85 @@
...
+    id = Stasiun Pengisian Mobil
+    it = Stazione di ricarica per autoveicoli
+    ja = 自動車充電ステーション
+    ko = 자동차 충전소
+    mr = मोटार कार चार्जिंग स्टेशन
+    nb = Ladestasjon for biler
+    nl = Oplaadstation voor auto's
+    pl = Stacja ładowania samochodów
+    pt = Estação de Carregamento de Automóveis
+    pt-BR = Estação de Carregamento de Automóveis
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:1120]: pt = Posto de carregamento de veículo elétrico
pt-BR = Eletroposto

**Review by matheusgomesms — CHANGES_REQUESTED** [review]
PT and PT-BR review

File: data/mapcss-mapping.csv (hunk 1 above)
Comment (pastk) [reply data/mapcss-mapping.csv:258]: I assume its the same charger for e-bikes, electric scooters, skate boards, etc.. Probably its just a power outlet?

**Pull request merged**